### DISCUSSION TITLE
Can the planner show original quantity not rounding up?

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#437] [Q&A]
I configure the operation material that adopted the gross die as 56.1, and executed a plan. I found the planner used 1 WF for fulfilling 43.4 dies. Just wondering know how can config that let the planner shows the original quantities as 0.773, not rounding up to 1?

**Comment by jdetaeye** [comment]
The operation modle has size-minimum, size-multiple and size-maximum fields to control the allowed size of the production orders.
By default size-minimum and size-multiple are set to force discrete quantities.  Set both of these to 0 to allow non-discrete quantities.
